Src/AutoFixture.NUnit3.UnitTest/AutoValueAttributeTest.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Moq;
+using NUnit.Framework;
+using NUnit.Framework.Interfaces;
+using TestTypeFoundation;
+
+namespace AutoFixture.NUnit3.UnitTest
+{
+    [TestFixture]
+    public class AutoValueAttributeTest
+    {
+        [Test]
+        [InlineAutoData(typeof(int))]
+        [InlineAutoData(typeof(string))]
+        [InlineAutoData(typeof(DoubleParameterType<int, string>))]
+        public void AutoValueAttribute_generates_a_model(Type typeToGenerate, AutoValueAttribute sut)
+        {
+            var mockParameter = new Mock<IParameterInfo>();
```

Comment:
@zvirja I added a dependency on Moq to quickly write this test. If you have better ways to solve this scenario, I'll make the proper changes.

I was too lazy to create a fake implementation of `IParameterInfo`.